### PR TITLE
[SECURITY] Use signed storage PID during frontend authentication

### LINKED ISSUE #104: Login not working since 10.4.33
Since the latest TYPO3 version released today, this extension doesn't seem to work anymore on our end.
Login procedure will not succeed.

### PR DESCRIPTION
**Pull request by chriwo**
Make the extension working with security fix of TYPO3 v10.4.33 and respect the new "security.frontend.enforceLoginSigning" feature flag (come with TYPO3 v10.4.34)

* TYPO3 security commit https://github.com/typo3/typo3/commit/96ed3e627f
* TYPO3 feature flag commit https://github.com/typo3/typo3/commit/0fe0322528

fixes #104

### CODE REVIEW

**Comment by mbrodala** [comment]
It would have been great if there was an API instead of having to copy and paste. :-/

File: Classes/Controller/AuthenticationController.php [hunk 1]
```diff
@@ -31,6 +31,7 @@
 use TYPO3\CMS\Extbase\Reflection\ObjectAccess;
 use TYPO3\CMS\Extbase\Security\Cryptography\HashService;
 use TYPO3\CMS\Extbase\SignalSlot\Dispatcher;
++use TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication;
```
Comment (mbrodala) [inline Classes/Controller/AuthenticationController.php:34]:
```suggestion
use TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication;
```

Comment (mbrodala) [reply Classes/Controller/AuthenticationController.php:34]: Can you fix this typo and push again? This should also trigger CI which was disabled in the meantime.

Comment (chriwo) [reply Classes/Controller/AuthenticationController.php]: Oh nooo, sorry my mistake and fixed now.

**Pull request merged**